frontend/tsconfig.json

```diff
@@ -7,8 +7,8 @@
     "moduleResolution": "bundler",
     // We use ESLint for this
     "allowUnreachableCode": true,
-    "jsx": "react",
+    "jsx": "react"
```

Comment:
@neilsong Something is wrong here. These commas should be there!

Reply:
Check master - it's a prettier update and I just did a clean install today and was really confused by this. I'm going to reconcile these branches in a second.